### DISCUSSION TITLE
32 bit binaries release for Linux

### DISCUSSION THREAD

**Discussion by GlassGruber** [BurntSushi/ripgrep#1996] [General] [answered]
Hei maybe I missed something, I see Windows 32bit binary are deployed, but not for Linux.

Is ripgrep still supporting 32 bit for Linux OS? Release 11.0.2 is the last available with 32bit binary.

Thank you! 🎩

**Comment by BurntSushi** [comment, marked as answer]
Support? Yes: https://github.com/BurntSushi/ripgrep/blob/9b01a8f9ae53ebcd05c27ec21843758c2c1e823f/.github/workflows/ci.yml#L63

But I don't publish binaries for it. You'll have to compile it yourself.

To be clear, this isn't about "32 bit linux." This is about "i686 linux."

**Reply by GlassGruber** [reply]
I see, thank you for your answer!
And you are right, I've (ab)used very generic terms, I've renamed the topic to partly address this. 

Out of curiosity you decided to provide 32bit binaries for Windows because of still relevant demand?

**Reply by BurntSushi** [reply]
I suspect i686 was giving me some trouble, and I do indeed perceive very low demand for it. I don't remember what trouble. In contrast, my understanding is that i686 Windows is still fairly popular.
